src/components/PlayerInteractions/PlayerInteractionsActions.js

```diff
@@ -0,0 +1,33 @@
+import { types } from './PlayerInteractionsActionTypes';
+import axios from "axios";
+import { removeCard } from '../MyCards/MyCardsActions';
+
+let instance = axios.create({ validateStatus: status => (status >= 200 && status < 500) });
+
+export const endTurn = () => (dispatch, getState) => {
+    const cardId = getState().myCardsReducer.playedCardId;
+    const word = getState().playWordReducer.word;
```

Comment:
Rather than calling `getState` twice, would it be better to assign an intermediate value?
```
const state = getState();
const cardId = state.myCardsReducer.playedCardId;
const word = state.playWordReducer.word;
```